test/NuGet.Tests.Apex/NuGet.Tests.Apex/Apex/NuGetApexTestService.cs

```diff
@@ -169,5 +173,27 @@ public NuGetConsoleTestExtension GetPackageManagerConsole(string project)
             var pmconsole = NuGetApexConsoleTestService.GetApexTestConsole();
             return new NuGetConsoleTestExtension(pmconsole, project);
         }
+
+        public bool EnsurePackageManagerConsoleIsOpen()
+        {
+            IVsWindowFrame window = null;
+            var powerConsoleToolWindowGUID = new Guid("0AD07096-BBA9-4900-A651-0598D26F6D24");
+            var stopwatch = Stopwatch.StartNew();
+            var timeout = TimeSpan.FromSeconds(10);
```

Comment:
Increase this timeout to 5 minutes or more unless you know that no test vm could ever be that slow